### ISSUE TITLE
Latency(bg) tests getting rejected when packet-interval set

### ISSUE THREAD

**Issue by arlake228** [perfsonar/pscheduler#818]
This is solely in the 4.2.0 branch and was able to confirm on multiple installs.

If you run a latency or latencybg and specify a "packet-interval" it fails with something like the following:

```
pscheduler task latency --source ps-dev-netlab-1.es.net --dest newy-owamp.es.net -i 0.1
Submitting task...
Unable to determine the lead participant: At /packet-interval: 0.1 is less than or equal to the minimum of True
```

The limits file on the hosts I tried do not reference packet-interval at all. There have been no changes to the validation or participants scripts so the best I can guess is one of the following:

1. Something changed in the json schema library and it is not properly checking the minimum value set in the schema. That value does set the `exclusiveMinimum` boolean so that may be confusing things?
2. The limits system is doing something goofy...which I kinda doubt since packet-interval is not referenced anywhere and I have not seen this type of issue anywhere else.

Debug logs for reference:

```
Apr 19 13:29:16 ps-dev-gridftp1 journal: pscheduler-api DEBUG    REQUEST: GET https://localhost/pscheduler/tests/latency/participants?spec={"dest"%3A+"newy-owamp.es.net"%2C+"source"%3A+"ps-dev-gridftp1.es.net"%2C+"packet-interval"%3A+0.1%2C+"schema"%3A+1}
Apr 19 13:29:16 ps-dev-gridftp1 journal: pscheduler-api DEBUG    Response 400: At /packet-interval: 0.1 is less than or equal to the minimum of True
```

**Comment by arlake228** [comment]
This was caused by a change in the JSON schema spec. We updated python-jsonschema to 3.0.1 which implements draft 7 of the JSON schema specification. In the version of the spec the exclusiveMinimum and exclusiveMaximum values are numeric values instead of booleans. This change caused the error above. The only tests using these specification values were latency and latencybg, so the schema docs have been updated to work with the latest definitions.
